internal/database/probe.go

```diff
@@ -0,0 +1,102 @@
+package database
+
+import (
+	"encoding/binary"
+	"encoding/json"
+	"fmt"
+
+	"github.com/boltdb/bolt"
+	"github.com/madjlzz/madprobe/internal/model"
+)
+
+type probeDto struct {
```

Comment:
It's just about terminology but I prefer to use `entity` instead of `dto`. Do you agree ?

Reply:
Yes it is better :)